### PR TITLE
Update extension README file

### PR DESCRIPTION
**Pull request by degrammer**

### CODE REVIEW

**Comment by sourishkrout** [comment]
@joseeantonior can you please go ahead and do a general content/lang review please?

File: README.md [hunk 1]
```diff
@@ -43,7 +61,7 @@ For instance, the following annotation will run the command in the background, g
 
 RUNME does not just retain frontmatter, it supports top-level settings that will affect all cells in the document.
```
Comment (joseeantonior) [inline README.md:62]: RUNME or Runme?

**Review by joseeantonior — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,8 +1,22 @@
-# [Runme](http://runme.dev) for [Visual Studio Code](https://code.visualstudio.com/) [![Tests](https://github.com/stateful/vscode-runme/actions/workflows/test.yml/badge.svg)](https://github.com/stateful/vscode-runme/actions/workflows/test.yml)
+---
+runme:
```
Comment (sourishkrout) [inline README.md:2]: @degrammer remove frontmatter please

**Review by sourishkrout — CHANGES_REQUESTED** [review]
No frontmatter in this file please (it's going on the MSFT marketplace). Otherwise 👍.

**Review by sourishkrout — APPROVED** [review]
👍 LGTM

**Pull request merged**